### ISSUE TITLE
[BUG] Windows installer for 1.6.0 blocked by Defender due to Trojan:Win32/Fuery.C!cl

### ISSUE THREAD

**Issue by jinxremoving** [Zettlr/Zettlr#842]
## Description
The Zettler 1.6.0 Windows installer was scanned and blocked by Windows Defender:
* Antimalware Client Version: 418.20046 
* Engine Version: 1.1.17000.7 
* Antivirus Version: 1.315.1414.0 
* Antispyware Version: 1.315.1414.0

## Reproducing
1. Download Zettler 1.6.0 from [here](https://github.com/Zettlr/Zettlr/releases/download/v1.6.0/Zettlr-1.6.0.exe)
2. Scan download with Windows Defender (if not automatic)
**Observed**: 
![image](https://user-images.githubusercontent.com/9146427/82863383-ddd2ef80-9ed6-11ea-84f3-3e48d9b364c9.png)

## Expected behaviour
Zettlr installer passes virus and anti-malware scans

## Platform
 - OS and version: Windows 10, 1909 (OS Build 18363.836)
 - Zettlr Version: 1.6.0
 - Screen Resolution: 2560x1600

**Comment by nathanlesage** [comment]
Would you mind dragging it through VirusTotal? I have a feeling that this one company may have not quite fixed their bug.

**Comment by pcrockett-pathway** [comment]
I had the same problem, but [VirusTotal says it's ok][1]. This isn't the first time I've had Windows Defender show me a false-positive recently.

I'm willing to bet that [using a code signing certificate would make these sorts of false positives less likely to happen][2]. That would allow Zettlr to get into Microsoft's software "reputation" system. That costs money though. :(

Alternatively, it looks like a Zettlr developer can submit the installer EXE to the [Windows Defender Security Intelligence portal][3] to help fix false positives.

[1]: https://www.virustotal.com/gui/url/ba9c7f6c3ec782e57c8d3d7fa8a59795fbae869adf2a65fed47a7d49860b1348/detection
[2]: https://www.microsoft.com/security/blog/2018/08/16/partnering-with-the-industry-to-minimize-false-positives/
[3]: https://www.microsoft.com/en-us/wdsi/filesubmission

**Comment by nathanlesage** [comment]
> That would allow Zettlr to get into Microsoft's software "reputation" system. That costs money though. :(

I already threw $300 to SSL.com for a certificate, and the current betas are code-signed — but not the 1.6. So reputation growth will begin from 1.7 onwards.

> Alternatively, it looks like a Zettlr developer can submit the installer EXE to the Windows Defender Security Intelligence portal to help fix false positives.

Also thought about it but as I can't automate this I won't do it manually. Let's hope for 1.7!

**Comment by jimvine** [comment]
I have had the same notification from Windows Defender today. I have uploaded the 1.6.0 installer to the Microsoft site requesting they recheck it. The site gave me an option to do that as a home user, not needing any affiliation to the developer.

**Comment by jimvine** [comment]
I got a response from my Microsoft submission. Their "Final determination" was "Not malware". There were also comments:

> Analyst comments:
> 
> We have removed the detection. Please follow the steps below to clear cached detection and obtain the latest malware definitions.
> 
> 1. Open command prompt as administrator and change directory to c:\Program Files\Windows Defender
> 2. Run “MpCmdRun.exe -removedefinitions -dynamicsignatures”
> 3. Run "MpCmdRun.exe -SignatureUpdate"
> 
> Alternatively, the latest definition is available for download here: https://www.microsoft.com/en-us/wdsi/definitions

I tried following their steps. The `-removedefinitions` step failed with an error. I tried rescanning the installer and it said that the item had been skipped due to an exclusion. So, I don't know whether the situation is now resolved for all users, but it does at least look like Microsoft have accepted that the file is harmless, and hopefully that will feed into their assessments of future versions.

**Comment by nathanlesage** [comment]
Thanks for validating the installer! And let's hope that reputation will build quickly with the newly code-signed installers — if you want to help, the betas are already signed with the certificate, so they'll already count towards the goal of "reputation". I'm going to leave that issue up, as in 67 days it'll be automatically closed, but this way other users until then have a chance to actually see this issue and get help with this problem!
